modules/odc-monitoring-your-application-metrics.adoc

```diff
@@ -5,15 +5,17 @@
 [id="odc-monitoring-your-application-metrics_{context}"]
 = Monitoring your application metrics
 
-After you create applications in your project and deploy them, you can use the *Topology* view in the *Developer* perspective to see the metrics for your application.
+After you create applications in your project and deploy them, you can use the *Topology* view in the *Developer* perspective to see the alerts and metrics for your application. Critical and warning alerts for your application are indicated on the workload node in the *Topology* view.
 
 .Procedure
+To see the alerts for your workload:
 
-. In the *Topology* view, click the application to see the application details in the right panel.
-. Click the *Monitoring* tab to see the warning events for the application; graphs for CPU, memory, and bandwidth usage; and all the events for the application.
+. In the *Topology* view, click the workload to see the workload details in the right panel.
+. Click the *Monitoring* tab to see the critical and warning alerts fired for the application; graphs for metrics, such as CPU, memory, and bandwidth usage; and all the events for the application.
```

Comment:
s/alerts fired for/alerts for

Reply:
Firing is a state of an alert. I have added a note for the same, and also listed the states that are not displayed to make it clear.